### PR TITLE
Add confidential policy flags to feature-flags.schema.json

### PR DESCRIPTION
**Pull request by Tolyas**
Add confidential policy flags.

### CODE REVIEW

File: docs/feature-flags.schema.json [hunk 1]
```diff
@@ -233,6 +233,21 @@
...
+    },
+    "global.confidentialPolicy.URL": {
+      "type": "string",
+      "title": "URL of confidential policy webpage",
+      "default": "",
+    },
+    "global.confidentialPolicy.version": {
+      "type": "string",
+      "title": "Version of confidential policy",
+      "default": "",
```
Comment (meafmira) [inline docs/feature-flags.schema.json]: Are these 2 really feature flags?

Comment (Tolyas) [reply docs/feature-flags.schema.json]: Those flags are from requirements. Since we won't have a separate method on server for this.

Comment (nkt) [reply docs/feature-flags.schema.json]: You can determine that confidential policy is off based on `global.confidentialPolicy.URL == null` check.

Comment (Tolyas) [reply docs/feature-flags.schema.json]: Anyway those two are necessary for proper agreement check. Should I remove `global.confidentialPolicy.required`?

Comment (Tolyas) [reply docs/feature-flags.schema.json]: Done.

File: docs/feature-flags.schema.json [hunk 2]
```diff
@@ -233,6 +233,16 @@
       "title": "Activates Calls",
       "default": false,
       "description": "if true = calls are enable for users"
+    },
+    "global.confidentialPolicy.URL": {
+      "type": "string",
+      "title": "URL of confidential policy webpage",
+      "default": "",
```
Comment (nkt) [inline docs/feature-flags.schema.json]: default is null

File: docs/feature-flags.schema.json [hunk 3]
```diff
@@ -233,6 +233,16 @@
...
+    },
+    "global.confidentialPolicy.version": {
+      "type": "string",
+      "title": "Version of confidential policy",
+      "default": "",
```
Comment (nkt) [inline docs/feature-flags.schema.json]: default is null

File: docs/feature-flags.schema.json (hunk 2 above)
Comment (Tolyas) [reply docs/feature-flags.schema.json]: fxd

**Pull request merged**